### PR TITLE
correct default datatypes

### PR DESCRIPTION
**Pull request by PatieC**
Summary of this change:

Change column data types

CHANGELOG entry:

 .
### Changed
Change column data types

How to test this code via the UI:
Dataminer | Reports | List  Reports

### CODE REVIEW

File: reports/incentive.yml [hunk 1]
```diff
@@ -278,7 +278,7 @@
     :sequence_no: 9
     :caption: Total fruit
     :namespaced_name: 
-    :data_type: :string
+    :data_type: :numeric
```
Comment (jcsjcs) [inline reports/incentive.yml:281]: Did you use the Datamine | Manage | Report Admin menu to do this?

- `numeric` should be `number`.
- Also the format should be delimited.

File: reports/incentive_report_per_comm_and_pack.yml [hunk 2]
```diff
@@ -12,10 +12,18 @@
           basic_pack_codes.basic_pack_code as pack_code,
           day_night_or_custom as shift_type_code,
           count(cartons.id) AS total_cartons,
+   wage_levels.wage_level ,
```
Comment (jcsjcs) [inline reports/incentive_report_per_comm_and_pack.yml:15]: Indentation is wrong.

**Review by jcsjcs — CHANGES_REQUESTED** [review]

File: reports/incentive.yml (hunk 1 above)
Comment (PatieC) [reply reports/incentive.yml:281]: I will correct, I was doing it manually in the file.

Comment (PatieC) [reply reports/incentive.yml:281]: Does this apply to all the fields  I have changed to numeric

Comment (jcsjcs) [reply reports/incentive.yml:281]: Yes. Use the Dataminer function.

**Pull request merged**